### PR TITLE
Use arcade-sim icons.css :)

### PR DESCRIPTION
**Pull request by livcheerful**
fixes https://github.com/microsoft/pxt-arcade/issues/3925

the icons.css in the sim also has....semantic icons at the end. I can't figure out how to append that in our build, so I'm just gonna update icons.css in arcade-sim.

### CODE REVIEW

File: cli/cli.ts [hunk 1]
```diff
@@ -717,7 +717,6 @@ function targetFileList() {
     if (simDir() != "sim")
         lst = lst.concat(nodeutil.allFiles(path.join("sim", "public"), 5, true))
     pxt.debug(`target files (on disk): ${lst.join('\r\n    ')}`)
-    lst = lst.filter(f => !f.includes("icons.css")) // We want to pull icons.css from pxt-core, not the target
```
Comment (eanders-MS) [inline cli/cli.ts:720]: Cloud icon still there?

Comment (livcheerful) [reply cli/cli.ts:720]: yup with https://github.com/microsoft/pxt-arcade-sim/pull/50

Comment (eanders-MS) [reply cli/cli.ts:720]: Oh! Do we need to do anything special to bump pxt-arcade-sim?

Comment (livcheerful) [reply cli/cli.ts:720]: hmmmmmmm i think @riknoll knows? not sure what being an unpublished npm package means...

Comment (riknoll) [reply cli/cli.ts:720]: You need to manually do a bump (change the version in package.json and push a tag).

Then you update the version in pxt-arcade in three places: The package.json and in tow of the github actions builds. Here's an example PR: https://github.com/microsoft/pxt-arcade/pull/3034/files

**Review by eanders-MS — APPROVED** [review]

**Pull request merged**